### PR TITLE
Use sysfs instead of deprecated procfs IF for thermal, if available.

### PR DESCRIPTION
**Pull request by borospeti**

### CODE REVIEW

File: landscape/lib/sysstats.py [hunk 1]
```diff
@@ -104,17 +108,27 @@ class ThermalZone(object):
...
-                    self.temperature = line[12:].strip()
-                    try:
-                        value, unit = self.temperature.split()
-                        self.temperature_value = int(value)
-                        self.temperature_unit = unit
-                    except ValueError:
-                        pass
+            with open(temperature_path) as f:
+                line = f.readline()
+                if re.match(r'^\d+\s*$', line):
```
Comment (maxiberta) [inline landscape/lib/sysstats.py]: I'd rather use a `try: except` block around `int()` as in the `else:` codepath as the most common, non-breaking case should be much faster.

File: landscape/lib/sysstats.py [hunk 2]
```diff
@@ -104,17 +108,27 @@ class ThermalZone(object):
...
+                    self.temperature_value = int(line.strip()) / 1000
+                    self.temperature_unit = 'C'
+                    self.temperature = '{:.1f}{}'.format(
```
Comment (maxiberta) [inline landscape/lib/sysstats.py]: This is just a nitpick, but based on tests in `landscape/lib/tests/test_sysstats.py` I think there should be a whitespace between value and unit.

File: landscape/lib/sysstats.py (hunk 1 above)
Comment (borospeti) [reply landscape/lib/sysstats.py]: Good catch, fixed.

File: landscape/lib/sysstats.py (hunk 2 above)
Comment (borospeti) [reply landscape/lib/sysstats.py]: Fixed.

**Pull request merged**